console/src/main/java/org/eclipse/rdf4j/console/command/Sparql.java

```diff
@@ -33,8 +36,15 @@ public String getHelpShort() {
 	@Override
 	public String getHelpLong() {
 		return PrintHelp.USAGE 
-			+ "sparql <query>                       Evaluates the SPARQL query on the currently open repository.\n"
 			+ "sparql                               Starts multi-line input for large SPARQL queries.\n"
+			+ "sparql <query>                       Evaluates the SPARQL query on the currently open repository.\n"
+			+ "\n"
+			+ "sparql INFILE=\"infile.ext\"           Evaluates the query stored in a file.\n"
+			+ "    Supported extensions: .jsonld, .nt, .ttl, .xml\n"
```

Comment:
Aren't these rather unusual extensions for a file containing a sparql query? Why not simply allow any file and just assume it's a text file containing a SPARQL query?

Reply:
whoops, indeed, those are actually extensions for the _output_ of graph results, not for input...

Reply:
I've updated the help info